lib/httparty/request.rb

```diff
@@ -9,7 +9,8 @@ class Request #:nodoc:
       Net::HTTP::Head,
       Net::HTTP::Options,
       Net::HTTP::Move,
-      Net::HTTP::Copy
+      Net::HTTP::Copy,
+      Net::HTTP::Mkcol
```

Comment:
Can we add a trailing comma here so we don't end up with a diff like this next time something is added?